src/schema/core.yaml

```diff
@@ -126,6 +127,68 @@ enums:
       liquid:
       gas:
 
+  ChemicalEntityEnum:
+    permissible_values:
+      acetonitrile:
+        meaning: CHEBI:38472
+      alphaLP:
+        description: >-
+          A serine protease that hydrolyzes peptide bonds at the C-terminus of threonine, alanine, serine, and valine.
+      ammonium_acetate:
+        meaning: CHEBI:62947
+      ammonium_bicarbonate:
+        meaning: CHEBI:184335
+      Arg-C:
+        meaning: MS:1001303
+        description: >-
+          A cysteine protease that hydrolyzes peptide, ester, and amide bonds at the C-terminus of arginine and with 
+          lower efficiency, lysine.
+      Asp-N:
+        meaning: MS:1001304
+        description: >-
+          A zinc metalloendopeptidase that hydrolyzes peptide bonds at the N-terminus of aspartic acid.
+      chloroform:
+        meaning: CHEBI:35255
+      chymotrypsin:
+        meaning: MS:1001306
+        description: >-
+          A serine protease that hydrolyzes peptide bonds at the C-terminus of tryptophan, leucine, tyrosine, 
+          and phenylalanine.
+      formic_acid:
+        meaning: CHEBI:30751
+        description: >-
+          A carboxylic acid that is the simplest aliphatic carboxylic acid, comprising a hydrogen atom 
+          joined to the methyl group of methanol.
+      glucose:
+        meaning: CHEBI:17234
+        description: >-
+          Generally considered the most abundant monosaccharide in nature.
+      Glu-C:
+        meaning: MS:1001917
+        description: >-
+          A serine protease that hydrolyzes peptide and ester bonds at the C-terminus of aspartic acid or glutamic acid.
+      isopropyl_alcohol:
+        meaning: CHEBI:17824
+      Lys-C:
+        meaning: MS:1001309
+        description: >-
+          A serine protease that hydrolyzes peptide, ester, and amide bonds at the C-terminus of lysine.
+      Lys-N:
+        meaning: MS:1003093
+        description: >-
+          A metalloendopeptidase that hydrolyzes peptide bonds at the C-terminus of lysine.
+      methanol:
+        meaning: CHEBI:17790
+      trypsin:
+        meaning: MS:1001251
+        description: >-
+          A serine protease that hydrolyzes peptide bonds at the C-terminus of arginine and lysine.
+      water:
+        meaning: CHEBI:15377
+        description: >- 
+          Any form of water used in an experiment including deionized water, distilled water, and ultrapure water.
+          Protocol or materials and methods should be consulted for exact type of water used in an experiment.
```

Comment:
This description includes very useful content, but most or all of it should be moved to one or more comments.  I'l admit that ChEBI's definition doesn't make for very good cocktail conversation, but I would choose it over the one provided here

> An oxygen hydride consisting of an oxygen atom that is covalently bonded to two hydrogen atoms